crypto/compactcert/const.go

```diff
@@ -22,8 +22,8 @@ import (
 
 // HashType/ hashSize relate to the type of hash this package uses.
 const (
-	HashType = crypto.Sha512_256
-	HashSize = crypto.Sha512_256Size
+	CompactCertHashType = crypto.Sha512_256
```

Comment:
names inside a package shouldn't start with the name of the package. 
throughout the package they are tied to the package, and outside of the package they are written as: 
packagename.Name